### PR TITLE
HtmlEditor: add translation for 'Border Width' in Italian (T1250873)

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

**Review by Zedwag — DISMISSED** [review]

File: packages/devextreme/js/localization/messages/vi.json [hunk 1]
```diff
@@ -195,8 +195,8 @@
         "dxDataGrid-masterDetail": "Cell with details",
 
         "dxTreeList-ariaTreeList": "Tree list with {0} rows and {1} columns",
-        "dxTreeList-ariaSearchInGrid": "Search in the tree list",
         "dxTreeList-ariaExpandableInstruction": "Press Ctrl + right arrow to expand the focused node and Ctrl + left arrow to collapse it",
+        "dxTreeList-ariaSearchInGrid": "Search in the tree list",
```
Comment (ksercs) [inline packages/devextreme/js/localization/messages/vi.json:199]: excess change

File: packages/devextreme/js/localization/messages/es.json [hunk 2]
```diff
@@ -417,6 +417,7 @@
         "dxHtmlEditor-delete": "Delete",
         "dxHtmlEditor-border": "Border",
         "dxHtmlEditor-style": "Style",
+        "dxHtmlEditor-borderWidth": "Border Width",
```
Comment (ksercs) [inline packages/devextreme/js/localization/messages/es.json]: [for the future, no need to change now]
It's better to locate new variables in a "sorted" order. For example below we have "dxHtmlEditor-borderColor" variable => I expect new "dxHtmlEditor-borderWidth" variable is located just after this one

**Review by ksercs — COMMENTED** [review]
@nikkithelegendarypokemonster  add ticket ID to the name please
And "HtmlEditor" should not be split - it's a single word :)

File: packages/devextreme/js/localization/messages/es.json (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/es.json]: [ec57a58](https://github.com/DevExpress/DevExtreme/pull/27994/commits/ec57a5870ae3b9e9bd13f892e39dfebb892e289b)

File: packages/devextreme/js/localization/messages/vi.json (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/vi.json:199]: [6a66add](https://github.com/DevExpress/DevExtreme/pull/27994/commits/6a66addc623c5845f0ebe1d857d2bde709b1f575)

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/vi.json:199]: This is necessary... this is added by auto-generation of localization messages. once remove its cause some CI issues

image

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/vi.json:199]: https://github.com/DevExpress/DevExtreme/actions/runs/10697916763/job/29656375230?pr=27994#step:3:578

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/localization/messages/vi.json:199]: [be6adb7](https://github.com/DevExpress/DevExtreme/pull/27994/commits/be6adb77d42d7943cc6cf6f269925de143058e28)

Comment (ksercs) [reply packages/devextreme/js/localization/messages/vi.json:199]: okay, i see that it other files the order is how it's updated in this one.
Do idea why previously it was a specific order for this particular locale - maybe sloppiness in previous PRs. 
Anyway it's not so important to pay much attention for this

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**